core/src/subgraph/instance_manager.rs

```diff
@@ -508,7 +501,17 @@ where
                 block.clone(),
                 triggers,
             )
-            .map(move |block_state| (ctx, block_state, data_sources, runtime_hosts))
+            .and_then(move |block_state| {
+                match block_state.created_data_sources.len() > created_ds_count {
+                    false => Ok((ctx, block_state, data_sources, runtime_hosts)),
+                    true => Err(err_msg(
+                        "A dynamic data source, in the same block that it was created,
+                        attempted to create another dynamic data source.
+                        Let us know in graph-node issue #1105 that you hit this.",
```

Comment:
This is a great idea. Can we replace `Let us know in graph-node issue #1105 that you hit this` with `Let us know if you hit this on https://github.com/graphprotocol/graph-node/issues/1105`? That way people can click on the link in their terminals if they run their own Graph node.